scripts/get_mappings_profile_control_levels.py

```diff
@@ -0,0 +1,123 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2025 Red Hat, Inc.
+import os
+import sys
+from pickle import DICT
+from ssg.controls import ControlsManager
+from ssg.products import load_product_yaml, product_yaml_path
+from ssg.profiles import _load_yaml_profile_file, get_profiles_from_products
+
+import logging
+
+"""
+Description:
+    This module is designed to handle the mapping of CAC profile names,
+    their corresponding available control files, and the selected security levels.
+
+    It is particularly useful to determine which profiles are mapped when detecting
+    updates of CAC control files. We could according to the mapped profiles to update
+    the related component-definition.
+
+    The output is a dictionary containing information about the profile,
+    policy ID, and the associated security levels.
+How to use it:
+    python get_mappings_profile_control_levels.py '<product>' '<content_root_dir>'"
+    E.g., $python get_mappings_profile_control_levels.py "rhel10" "/path/of/cac-content"
+Output Format:
+    The module produces an output dictionary with the following structure:
+
+    {
+        'profile_name': 'stig',
+        'policy_id': 'stig_rhel9',
+        'levels': ['high', 'medium', 'low']
+    }
+"""
+
+logging.basicConfig(level=logging.INFO, format="%(message)s")
+
+
+def load_controls_manager(cac_content_root, product) -> ControlsManager:
```

Comment:
This function can move to [utils.py](https://github.com/complytime/trestle-bot/blob/main/trestlebot/utils.py) , i notice it's a common use case.

Reply:
Thanks, updated.